proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -63,6 +63,16 @@ message SignDoc {
   uint64 account_number = 4;
 }
 
+message SignDocAux {
+  bytes               body_bytes     = 1;
+  google.protobuf.Any public_key     = 2;
+  string              chain_id       = 3;
```

Comment:
We should add some sequence to avoid replay attacks

Reply:
Ah good point

Reply:
I feel like we end up needing a sequence numbers for both signers.